src/Components/AdvancedSearch/AdvancedSearchModal.tsx

```diff
@@ -89,9 +112,11 @@ const AdvancedSearchModal: React.FC<IAdvancedSearchProps> = (props) => {
                     </div>
                     <div className={classNames.resultsContainer}>
                         <AdvancedSearchResultDetailsList
```

Comment:
can we show "No results" text instead of showing the header of the table when the result is 0 item?
<img width="722" alt="image" src="https://user-images.githubusercontent.com/45217314/188195509-6f32b5da-0462-4301-a3dd-2a265a8969e9.png">

Also is there a reason why we made the Results title bold? or maybe we can use h4 instead of h3 what do you think @azimamansuri ?

Reply:
also I think the options in the dropdown are bolder now, but after selecting the input text is lighter weight. Is there a reason why we made the options bolder? What do you think @azimamansuri?:
<img width="302" alt="image" src="https://user-images.githubusercontent.com/45217314/188197080-5e517a26-df7c-489e-a297-3e1ab0998153.png">

Reply:
+1 on the no data message. 
Should probably have a different message before the query is fired too "Build a query to search the twins graph" or something.

Reply:
Added no data message for both scenarios